t/app/controller/my_planned.t

```diff
@@ -57,6 +57,31 @@ $mech->get_ok('/my/planned?ajax=1');
 $mech->content_contains('shortlist-up');
 $mech->content_contains('shortlist-down');
 
+subtest "POSTing multiple problems to my/planned/change adds all to shortlist" => sub {
+    my ($problem1, $problem2, $problem3) = $mech->create_problems_for_body(3, $body->id, 'New Problem');
+
+    # Grab CSRF token
+    $mech->get_ok($problem1->url);
+    my ($csrf) = $mech->content =~ /meta content="([^"]*)" name="csrf-token"/;
+
+    $mech->post_ok( '/my/planned/change_multiple', {
+            'ids[]' => [
```

Comment:
The IDs will be an array in the JS though, so server side, it's received as an array.